pkgs/tools/games/wljoywake/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, udev
+, wayland
+, wayland-protocols
+, wayland-scanner
+}:
+
+let
+  version = "0.3";
+in
+stdenv.mkDerivation {
+  pname = "wljoywake";
+  inherit version;
+
+  src = fetchFromGitHub {
+    repo = "wljoywake";
+    owner = "nowrep";
+    rev = "v${version}";
+    hash = "sha256-zSYNfsFjswaSXZPlIDMDC87NK/6AKtArHBeWCWDDR3E=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
+    udev
+    wayland
+    wayland-protocols
+    wayland-scanner
```

Comment:
wayland-scanner goes to nativeBuildInputs

Reply:
Done